pkgs/development/python-modules/cusfpredict/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  deprecated,
+  hopcroftkarp,
+  joblib,
+  matplotlib,
+  numpy,
+  scikit-learn,
+  scipy,
+  pytestCheckHook,
+  pythonAtLeast,
+  pythonOlder,
+}:
+buildPythonPackage rec {
+  pname = "cusfpredict";
+  version = "0.2.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "darksidelemm";
+    repo = "cusf_predictor_wrapper";
+    rev = "f4352834a037e3e2bf01a3fd7d5a25aa482e27c6";
+    hash = "sha256-C8/5x8tim6s0hWgCC7LpN1hesdVME5kpQFqDTEyXHtg=";
+  };
```

Comment:
Please share the source code with `cusf-predictor-wrapper`.
I think these should be combined into a single package.

Reply:
How would this work in practice? It's a monorepo but there seems to be two distinct projects in the repo, each with a separate build system. Can they be combined into one package? Furthermore, if you want one without the other, wouldn't you still need to build both if we combined them? Chasemapper itself only needs `cusfpredict`, not the wrapper